examples/python-django/devenv.nix

```diff
@@ -0,0 +1,44 @@
+{ pkgs, config, ... }:
+
+let
+  db_user = "postgres";
+  db_host = "localhost";
+  db_port = "5432";
+  db_name = "db";
+in
+{
+  packages = [ pkgs.git pkgs.postgresql_14 ];
+
+  languages.python = {
+    enable = true;
+    package = pkgs.python310;
+    poetry.enable = true;
+  };
+
+  env = {
+    DATABASE_URL = "postgres://${db_user}@/${db_name}?host=${config.env.PGHOST}";
+    DEBUG = true;
+    SECRET_KEY = "123";
+    STATIC_ROOT = "/tmp";
+  };
+
+  services.postgres = {
+    enable = true;
+    initialScript = "CREATE USER ${db_user} SUPERUSER;";
+    initialDatabases = [{ name = db_name; }];
+    listen_addresses = db_host;
+  };
+
+  processes = {
+    runserver.exec = ''
+      devenv shell python manage.py runserver
+    '';
+  };
+
+  scripts = {
+    enterTest.exec = ''
```

Comment:
This should be `enterTest = ''` without exec and scripts

Reply:
Thanks @domenkozar - that sorts it - 9bf74782888043494d49afd621df0deb0d8b7ea6